### ISSUE TITLE
Sans Bold Italic read as Sans Italic

### ISSUE THREAD

**Issue by SverreStausland** [libertinus-fonts/libertinus#210]
There is apparently a bug in the Sans Bold Italic font that makes XeLaTeX load it as the Sans Italic font, cf. https://tex.stackexchange.com/q/447029/9077

**Comment by alerque** [comment]
@khaledhosny Have the authors of the CTAN package ever tried to contribute their additional styles upstream?

**Comment by khaledhosny** [comment]
Yes. The extra style is automatically generated with FontForge, and I got asked if it is any good. I think it is good as a quick hack when faux bold or italic are not available, but if we are going to integrate it then it will need some serious cleanup and drawing from scratch might even be cleaner. That being said, I think bold italic is really overrated and I probably had to use it once or twice in my entire life.
